packages/babel-plugin-transform-classes/src/transformClass.js

```diff
@@ -0,0 +1,718 @@
+// @flow
```

Comment:
It looks like this file was given Flow annotations, but fails a few checks. Would someone be up for taking a look? Looks like just a couple failures.

Reply:
Oh, sorry about that! I wasn't sure how to type it in a usable way (without blindly patching) 😬

Reply:
No problem. I'd say we should just remove the `@flow` annotation if it isn't actually intended to typecheck them. The flow type definitions are still useful for documentation, so those are fine though.